index.html

```diff
@@ -148,6 +148,43 @@ <h1>Multi-Post Stories</h1>
  
             </article>
         </section>
+        <section class="contact-form">
+            <h3>CONTACT US</h3>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p> 

```

Comment:
- [ ] Again here kindly use the provided font in the Figma template as this is part of the core requirements.